### PR TITLE
docs(fr): contribution-guide.md

### PR DESCRIPTION
**Pull request by roiLeo**
French Translation:  
- [x] contribution-guide

### CODE REVIEW

File: content/fr/contribution-guide/index.md [hunk 1]
```diff
@@ -1,105 +1,105 @@
...
 
 ## Pull Requests
 
-We'd love to see your pull requests, even if it's just to fix a typo!
+Nous serons ravis de recevoir vos demandes de modification, même s'il s'agit simplement de corriger une coquille !
 
-However, any significant improvement should be associated to an existing [feature request](https://feature.nuxtjs.org/) or [bug report](https://bug.nuxtjs.org/).
+Cependant, toute amélioration significative doit être associée à une demande de [fonctionnalité](https://feature.nuxtjs.org/) ou d'un [bug report](https://bug.nuxtjs.org/).
 
 ### Getting started
```
Comment (smarroufin) [inline content/fr/contribution-guide/index.md:22]: I don't know the best words to translate this but something like "Démarrer"/"Bien démarrer"?

File: content/fr/contribution-guide/index.md [hunk 2]
```diff
@@ -1,105 +1,105 @@
...
 
-1. [Fork](https://help.github.com/articles/fork-a-repo/) the [Nuxt repository](https://github.com/nuxt/nuxt.js) to your own GitHub account and then [clone](https://help.github.com/articles/cloning-a-repository/) it to your local device.
-2. Run `npm install` or `yarn install` to install the dependencies.
+1. [Dupliquez (Fork)](https://help.github.com/articles/fork-a-repo/) le [dépôt Nuxt](https://github.com/nuxt/nuxt.js) sur votre propre compte GitHub, puis le [cloner](https://help.github.com/articles/cloning-a-repository/) sur votre machine local.
```
Comment (smarroufin) [inline content/fr/contribution-guide/index.md:24]: I'd say "machine local**e**" or "machine en local"

File: content/fr/contribution-guide/index.md [hunk 3]
```diff
@@ -1,105 +1,105 @@
...
+> _Notez que **npm** et **yarn** ont été vus comme manquant l'installation des dépendances. Pour remédier à cela, vous pouvez soit supprimer le dossier `node_modules` de votre application d'exemple et l'installer à nouveau, soit faire une installation locale des dépendances manquantes._
 
-> If you are adding a dependency, please use `yarn add`. The `yarn.lock` file is the source of truth for all Nuxt dependencies.
+> Si vous ajoutez une dépendance, veuillez utiliser `yarn add`. Le fichier `yarn.lock` est la source de vérité pour toutes les dépendances de Nuxt.
 
-### Setup
+### Installation
 
-Before running any tests, make sure all dependencies are met and build all packages:
+Avant d'exécuter les tests, assurez-vous que toutes les dépendances ont bien installées et vous pouvez alors construire les paquets :
```
Comment (smarroufin) [inline content/fr/contribution-guide/index.md:33]: "ont bien **été** installées"

File: content/fr/contribution-guide/index.md [hunk 4]
```diff
@@ -1,105 +1,105 @@
...
 ```sh
 yarn
 yarn build
 ```
 
-### Test structure
+### Structure d'un Test
 
-A great PR, whether it includes a bug fix or a new feature, will often include tests. To write great tests, let us explain our test structure:
+Une bonne Pull Request, qu'il s'agisse d'une correction de bogue ou d'une nouvelle fonctionnalité, comprendra souvent des tests. Pour écrire de bons tests, nous allons vous expliquer la structure attendu :
```
Comment (smarroufin) [inline content/fr/contribution-guide/index.md:42]: "la structure attendu**e**"

File: content/fr/contribution-guide/index.md [hunk 5]
```diff
@@ -1,105 +1,105 @@
...
 #### Fixtures
 
-The fixtures (found under `tests/fixtures`) contain several Nuxt applications. To keep build time as short as possible, we don't build an own Nuxt application per test. Instead, the fixtures are built (`yarn test:fixtures`) before running the actual unit tests.
+Les fixtures (trouvés sous `tests/fixtures`) contiennent plusieurs applications Nuxt. Pour que le temps de construction soit le plus court possible, nous ne construisons pas une application Nuxt par test. Au lieu de cela, les fixtures sont construites (`yarn test:fixtures`) avant d'exécuter les tests unitaires.
 
-Please make sure to **alter** or **add a new fixture** when submitting a PR to reflect the changes properly (if applicable).
+Veillez à **modifier** ou **ajouter une nouvelle fixture** lorsque vous soumettez une PR afin de refléter correctement les changements (si applicable).
 
-Also, don't forget to **rebuild** a fixture after changing it by running the corresponding test with `jest test/fixtures/my-fixture/my-fixture.test.js`!
+N'oubliez pas non plus de **reconstruire** un fixture après l'avoir modifié en exécutant le test correspondant avec `jest test/fixtures/my-fixture/my-fixture.test.js`!
```
Comment (smarroufin) [inline content/fr/contribution-guide/index.md:50]: I would say "une fixture". So 
- "trouvé**e**s"
- "reconstruire un**e** fixture après"

File: content/fr/contribution-guide/index.md [hunk 6]
```diff
@@ -1,105 +1,105 @@
...
 ```
 
-Again, please be aware that you might have to rebuild your fixtures before!
+Encore une fois, sachez qu'il se peut que vous deviez reconstruire vos fixtures avant !
 
-### Testing your changes
+### Tester vos changements
 
-While working on your PR you will likely want to check if your fixture is set up correctly or debug your current changes.
+Pendant que vous travaillez sur votre PR, vous voudrez probablement vérifier si votre fixture est configuré correctement ou déboguer vos modifications en cours.
```
Comment (smarroufin) [inline content/fr/contribution-guide/index.md:72]: "votre fixture est configuré**e**"

File: content/fr/contribution-guide/index.md [hunk 7]
```diff
@@ -1,105 +1,105 @@
...
 
 ```sh
 yarn nuxt examples/your-app
 yarn nuxt test/fixtures/your-fixture-app
 ```
 
-> `npm link` could also (and does, to some extent) work for this, but it has been known to exhibit some issues. That is why we recommend calling `yarn nuxt` directly to run examples.
+> `npm link` pourrait également fonctionner pour cela (et fonctionne, dans une certaine mesure), mais il est connu pour présenter quelques problèmes. C'est pourquoi nous recommandons d'appeler directement `yarn nuxt` pour exécuter les exemples.
 
 ### Examples
```
Comment (smarroufin) [inline content/fr/contribution-guide/index.md:83]: "Exemples"

**Review by smarroufin — CHANGES_REQUESTED** [review]

**Comment by roiLeo** [comment]
@smarroufin Thanks for the quick review! Added some changes.

**Review by smarroufin — APPROVED** [review]
LGTM! Thanks for your contribution.
May be wait for a second pair of eyes and we are good.

**Review by Atinux — APPROVED** [review]

**Pull request merged**